dpnp/linalg/dpnp_utils_linalg.py

```diff
@@ -2277,22 +2439,20 @@ def dpnp_svd(
         u_shape,
         dtype=uv_type,
         order="F",
-        usm_type=a_usm_type,
-        sycl_queue=a_sycl_queue,
+        usm_type=usm_type,
```

Comment:
guessing everywhere here we can use `empty_like`, because we have the same `order`, `usm_type` and `sycl_queue` as `a_h`